test/Makefile

```
@@ -30,6 +31,26 @@ run:
 k8s:
 	ginkgo --focus "K8s" --tags integration_tests -v -- --cilium.provision=$(PROVISION) --cilium.registryCredentials=$(REGISTRY_CREDENTIALS)
 
+# Match kind-image target in parent directory
+k8s-kind: export DOCKER_REGISTRY=localhost:5000
+k8s-kind:
+	@if [ -z $(FOCUS) ]; then \
+		>&2 echo "usage: FOCUS=K8sFoo make k8s-kind"; \
+		exit 1; \
+	fi
+	@CNI_INTEGRATION=kind \
+		K8S_VERSION="$$(kubectl version -o json | jq -r '.serverVersion | "\(.major).\(.minor)"')" \
+		ginkgo --focus "$(FOCUS)" --tags integration_tests -v -- \
+			-cilium.testScope=k8s \
+			--cilium.provision=false \
```

I think this should be
```suggestion
			-cilium.provision=false \
```